docs/download_files.md

```diff
@@ -68,3 +70,8 @@ The `libraries_metadata.csv` file is a comma-separated table with one row per li
 
 Additional metadata may also be included, specific to the disease type and experimental design of the project.
 Examples of this include treatment or outcome.
+Metadata pertaining to processing will also be available in this table and inside of the `SingleCellExperiment` object.
+See the {ref}`Experiment metadata <sce_file_contents:experiment metadata>` section for more information on metadata columns that can be found in this file as well as inside the `SingleCellExperiment` object.
+
+For projects with bulk RNA-seq, the `bulk_metadata.tsv` file will be included for project downloads. 
+This file will contain all of the fields found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).
```

Comment:
Removing "all of" in case it is not always true.
```suggestion
This file will contain fields equivalent to those found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).
```